src/main/java/de/hglabor/bedwars/Bedwars.java

```diff
@@ -25,6 +27,8 @@ public void onEnable() {
         getCommand("settings").setExecutor(new SettingsCommand());
         Settings.addSetting("testFloatSetting", new FloatSetting("Test FloatSetting", 5f, 0f, 23f, Criteria.COMBAT));
         Settings.addSetting("testBooleanSetting", new BooleanSetting("Test BooleanSetting", false, Criteria.MAP));
+        Settings.addSetting("testIntSetting", new IntSetting("Test IntSetting", 5, 0, 12, Criteria.MECHANICS));
+        Settings.addSetting("testEnumSetting", new EnumSetting<>("Test EnumSetting", Criteria.GENERIC, Criteria.GENERIC));
```

Comment:
EnumSetting isn't done yet, it requires some reflection and i don't know how to visualize a Enum into an inventory

Reply:
Ich würde sagen, dass man anzeigt, was bei Rechtsklick und Linksklick kommt.
z.b.
    <- ENTRY1
         ENTRY3 ->
Soll ich das wieder rausnehmen oder passt das so fürs erste?

Reply:
jo so oder 2. inv mit allen enum constants öffnen

nimm erstmal raus

Reply:
Habs wieder rausgenommen.
Hast Recht, man sollte ein neues Fenster aufmachen